plugins/syslog_logs.yaml

```diff
@@ -3,34 +3,43 @@ title: Syslog
 description: Log receiver for Syslog
 parameters:
   - name: connection_type
+    description: Transport protocol to use (`udp` or `tcp`)
     type: string
     supported:
       - tcp
       - udp
     default: udp
   - name: protocol
+    description: Protocol of received syslog messages (`rfc3164 (BSD)` or `rfc5424 (IETF)`)
     type: string
     supported:
       - rfc3164
       - rfc5424
     default: rfc5424
-  - name: location
-    type: string
+  - name: timezone
+    description: Timezone to use when parsing the timestamp
+    type: timezone
     default: UTC
   - name: listen_address
+    description: Parameter Deprecated Use 'listen_ip' and 'listen_port' instead
```

Comment:
I don't see a `listen_ip` and `listen_port` in this plugin. We likely should change this over to that format at some point but right now this description is asking the user to use parameters that don't exist.